frontend/sw360-portlet/src/main/java/org/eclipse/sw360/portal/portlets/Sw360Portlet.java

```diff
@@ -263,6 +263,8 @@ public void setSessionMessage(PortletRequest request, RequestStatus requestStatu
                     setSW360SessionError(request, ErrorMessages.DOCUMENT_USED_BY_PROJECT_OR_RELEASE);
                 }
                 break;
+            case FAILED_SANITY_CHECK:
+                setSW360SessionError(request, ErrorMessages.UPDATE_FAILED_SANITY_CHECK);
```

Comment:
A `break;` is missing at this place causing the following `PortletException` to be thrown.